### PR TITLE
[lab_2] Add script for deploing backend and frontend

### PR DESCRIPTION
**Pull request by Ksarelto**
Task name: Lab 2: Networking & Security. Web Servers.
Task link: https://github.com/EPAM-JS-Competency-center/devops-fundamentals-course/tree/main/lab_2

### CODE REVIEW

File: module_8/frontend/frontend_shop.conf [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		proxy_pass http://127.0.0.1:8080/; 
+	}
+}
+
+server {
+  listen 80;
+  listen [::]:80;
+  server_name react-shop.com; 
+
+  location / {
```
Comment (KunskayaV) [inline module_8/frontend/frontend_shop.conf]: FYI: you can add this rule for all requests that your server receives without any location:

```
server {
  listen 80;
  server_name react-shop.com;
  return 301 https://$host$request_uri;
}
```

https://www.digitalocean.com/community/tutorials/nginx-rewrite-url-rules

File: module_8/backend/src/services/config/configuration.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export default () => ({
```
Comment (KunskayaV) [inline module_8/backend/src/services/config/configuration.ts]: don't really see how this file is used. Can you pls clarify a bit?

**Review by KunskayaV — COMMENTED** [review]
Pls provide local conf file.
I don't see backend upstream configured on screenshots provided for local file.
It will be more clear if it's in this PR as well

File: module_8/frontend/frontend_shop.conf (hunk 1 above)
Comment (Ksarelto) [reply module_8/frontend/frontend_shop.conf]: fixed

File: module_8/backend/src/services/config/configuration.ts (hunk 2 above)
Comment (Ksarelto) [reply module_8/backend/src/services/config/configuration.ts]: I forget delete it. Fixed

**Review by KunskayaV — APPROVED** [review]
LGTM

**Pull request merged**